lib/model/store.dart

```diff
@@ -235,6 +235,7 @@ class PerAccountStore extends ChangeNotifier with EmojiStore, ChannelStore, Mess
       connection: connection,
       realmUrl: realmUrl,
       maxFileUploadSizeMib: initialSnapshot.maxFileUploadSizeMib,
+      realmWaitingPeriodThreshold: initialSnapshot.realmWaitingPeriodThreshold,
```

Comment:
Moved it before the line for `maxFileUploadSizeMib`.

Also, looking at these "realm and server" section properties, I'm unsure if they follow a specific order. If they're meant to follow the order of `InitialSnapshot`, then they'll have an order like this:
```dart
customProfileFields
realmWaitingPeriodThreshold
realmUrl
realmDefaultExternalAccounts
maxFileUploadSizeMib
emailAddressVisiblity
```

Reply:
The constructor arguments at this call site should match the order of the parameters in the constructor's declaration, which in turn should match the order the fields are declared in.

For the field declarations, generally we've tried to keep them in a logical order (which is our default approach for ordering things). So `realmUrl` comes before all the miscellaneous settings, because it's the value that identifies the whole realm in the first place. I don't think there's a strong reason the other five you listed are ordered in the way they currently are, though.

That logical ordering may differ from what's in `InitialSnapshot`, because:
```dart
class InitialSnapshot {
  // Keep these fields in the order they appear in the API docs.
  // (For many API types we choose a more logical order than the docs.
  // But this one is so long that that'd make it become impossible to
  // compare the lists by hand.)
```